### DISCUSSION TITLE
wlan card compatibility

### DISCUSSION THREAD

**Discussion by BloodLancet** [valnoxy/t480-oc#108] [Q&A]
do t480/t580 have wlan card whitelist? i'm planning on buying bcm card but don't know which one to buy

**Comment by rivalscott04** [comment]
as far as i know its not, i am using t480s and i am change my wifi card to bcm94360

**Comment by bobmarley2021** [comment]
There is definitely no whitelist. I have tried several cards under Windows before Hackintoshing and all worked.
